### PR TITLE
ima-plugin-analytic deferred loading of analytic script

### PR DESCRIPTION
**Pull request by VojtaSim**

### CODE REVIEW

File: packages/plugin-analytic-google/src/GoogleAnalytic.js [hunk 1]
```diff
@@ -95,11 +89,7 @@ export default class GoogleAnalytic extends AbstractAnalytic {
   _configuration() {
     const clientWindow = this._window.getWindow();
 
-    if (
-      !clientWindow.ga ||
-      typeof this._window.getWindow().ga !== 'function' ||
-      this.isEnabled()
-    ) {
+    if (!clientWindow.ga || typeof this._window.getWindow().ga !== 'function') {
```
Comment (mjancarik) [inline packages/plugin-analytic-google/src/GoogleAnalytic.js]: why did you remove isEnabled()?

Comment (VojtaSim) [reply packages/plugin-analytic-google/src/GoogleAnalytic.js]: Because the `_configuration` method won't be even executed if `this.isEnabled()` returns `true`.
See https://github.com/seznam/IMA.js-plugins/pull/6/files#diff-5a1a4fc5265fa9626cc10c3db3756d1aR86

File: packages/plugin-analytic-google/src/GoogleAnalytic.js [hunk 2]
```diff
@@ -54,19 +52,17 @@ export default class GoogleAnalytic extends AbstractAnalytic {
...
-          data.method || 'send',
-          data.type || 'event',
-          data.category || 'undefined',
-          data.action || 'undefined',
-          data.label,
-          data.value,
-          data.fields
-        );
-    });
+    this._window
```
Comment (mjancarik) [inline packages/plugin-analytic-google/src/GoogleAnalytic.js]: The method is not safe for server side because getWindow returns undefined. The last version before implementation of defer hit was safe https://github.com/seznam/IMA.js-plugins/blob/bf729e6e8d761c6fd914ee62599b1b227620a01f/packages/plugin-analytic-google/src/GoogleAnalytic.js#L74.

Comment (mjancarik) [reply packages/plugin-analytic-google/src/GoogleAnalytic.js]: All public methods must be safe for server side.

Comment (VojtaSim) [reply packages/plugin-analytic-google/src/GoogleAnalytic.js]: Utilized `this.isEnabled()` method in `hit` and `hitPageView` methods before requesting window object.
